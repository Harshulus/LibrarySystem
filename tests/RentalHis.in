add collection SampleCollection.csv
add collection SampleCollection.csv
add member asvi
add member bob
member history 100000
rent 100000 111111
rent 100000 111115
relinquish 100000 111115
relinquish 100000 111111
member history 100002
member history 100001
member history 100000
