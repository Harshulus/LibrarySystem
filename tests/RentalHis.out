user: 17 books successfully added.

user: 17 books successfully added.

user: Success.

user: Success.

user: No rental history for member.

user: Success.

user: Success.

user: Success.

user: Success.

user: No such member in system.

user: No rental history for member.

user: The Hitchhiker's Guide to the Galaxy (Douglas Adams)
To Kill a Mockingbird (Harper Lee)

